tabs/osd.js

```diff
@@ -488,31 +485,37 @@ TABS.osd.initialize = function (callback) {
             });
 
             // units
-            var $unitMode = $('.units').empty();
-            for (var i = 0; i < OSD.constants.UNIT_TYPES.length; i++) {
-              var type = OSD.constants.UNIT_TYPES[i];
-              var $checkbox = $('<label/>').append($('<input name="unit_mode" type="radio"/>'+type+'</label>')
-                .prop('checked', i === OSD.data.unit_mode)
-                .data('type', type)
-                .data('type', i)
-              );
-              $unitMode.append($checkbox);
-            }
-            $unitMode.find(':radio').click(function(e) {
-              OSD.data.unit_mode = $(this).data('type');
-              MSP.promise(MSPCodes.MSP_SET_OSD_CONFIG, OSD.msp.encodeOther())
-              .then(function() {
-                updateOsdView();
+            if (semver.gte(CONFIG.flightControllerVersion, "3.0.1")) {
+              $('.units-container').show();
+              var $unitMode = $('.units').empty();
+              for (var i = 0; i < OSD.constants.UNIT_TYPES.length; i++) {
+                var type = OSD.constants.UNIT_TYPES[i];
+                var $checkbox = $('<label/>').append($('<input name="unit_mode" type="radio"/>'+type+'</label>')
+                  .prop('checked', i === OSD.data.unit_mode)
+                  .data('type', type)
+                  .data('type', i)
+                );
+                $unitMode.append($checkbox);
+              }
+              $unitMode.find(':radio').click(function(e) {
+                OSD.data.unit_mode = $(this).data('type');
+                MSP.promise(MSPCodes.MSP_SET_OSD_CONFIG, OSD.msp.encodeOther())
+                .then(function() {
+                  updateOsdView();
+                });
               });
-            });
+            }
 
             // display fields on/off and position
             var $displayFields = $('.display-fields').empty();
-            for (var field in OSD.data.display_items) {
+            for (let field of OSD.data.display_items) {
```

Comment:
I'm not in favour of using language constructs that are outside of the JavaScript version that is officially supported by chrome. This makes the use of IDEs to check for correctness pretty much meaningless.